### DISCUSSION TITLE
Hacktoberfest 2021

### DISCUSSION THREAD

**Discussion by T1mL3arn** [HaxeFlixel/flixel#2432] [General]
Though it is 2/3 complete, HaxeFlixel can still participate in [Hacktoberfest 2021](https://hacktoberfest.digitalocean.com) as a maintainer. In short - you just need to add `Hacktoberfest` topic to all repositories which you want to participate.

What it is

>Hacktoberfest, in its 8th year, is a month-long celebration of open source software run by DigitalOcean. During the month of October, we invite you to join open-source software enthusiasts, beginners, and the developer community by contributing to open-source projects. You can do this in a variety of ways

More details about what it is, detailed rules and recommendations can be found in the link above.

**Comment by RichardBray** [comment]
Yeah, I think it's a good idea. Although I don't know if the maintainers want a bunch of randos correcting their typos 😁 

Nevertheless, if you're interested in fixing HF bugs I would hope that your love for the community and engine should be better than a free t-shirt 👕 😄

**Reply by T1mL3arn** [reply]
This year the event handles in a better way, that is why it is needed to add a topic to repos and also review PR's, so typo-fixing PR's can be dismissed. There is also a possibility to count a single PR by marking it with label `hacktoberfest-accepted`. Full recommendations for maintainers can be found [there](https://hacktoberfest.digitalocean.com/resources/maintainers).

**Reply by RichardBray** [reply]
Ah cool, didn't know that was the case, that makes it much better than last year.

**Comment by Geokureli** [comment]
> In short - you just need to add Hacktoberfest topic to all repositories which you want to participate
Sorry if this is a dumb question but where does that get added? Does this topic count as a topic named `Hacktoberfest`?

**Reply by T1mL3arn** [reply]
Topics - is how github names repository tags. For HaxeFlixel topics (or tags) are:

![image](https://user-images.githubusercontent.com/9349164/138473619-b7086c8b-6240-4a59-a6ad-3ac7806b6124.png)

If you are a maintainer/owner, there will be a gear button to change topics.

**Reply by Geokureli** [reply]
@Gama11, what do you think? seems like a good idea to me
